chatlib/src/main/java/io/mrarm/chatlib/irc/cap/SASLCapability.java

```diff
@@ -54,6 +56,11 @@ public void handle(ServerConnectionData connection, MessagePrefix sender, String
             if (params.size() == 1 && CommandHandler.getParamWithCheck(params, 0).equals("+"))
                 continueAuthentication(connection);
         } else if (numeric == RPL_SASLSUCCESS) {
+            String message = CommandHandler.getParamWithCheck(params, 1, "SASL authentication successful");
```

Comment:
This should be set to null as the string is not translatable. Then you can do a fall back in the UI to "SASL authentication successful" here: https://github.com/MCMrARM/revolution-irc/blob/a277059a964d607ea95315170b95bcda2c2d9578/app/src/main/java/io/mrarm/irc/util/MessageBuilder.java#L409-L455 (first add the string to strings.xml then use `mContext.getString(R.string.message_sasl_successful)`).